scss/pager/_theme.scss

```diff
@@ -79,6 +79,11 @@
 
     }
 
+    .k-link.k-pager-nav.k-state-disabled {
+        border-color: transparent;
+        background-color: transparent;
```

Comment:
Disabled grid pager buttons are hidden after this update:

| Before | After |
|-|-|
| ![image](https://user-images.githubusercontent.com/90405/31772507-96ac8fa6-b4e8-11e7-879f-8559acab8d2a.png) | ![image](https://user-images.githubusercontent.com/90405/31772488-88ccb6d6-b4e8-11e7-816b-cee47c648e1c.png) |

This appears to be intentional, given these lines, but differs from the Bootstrap styles:

![image](https://user-images.githubusercontent.com/90405/31772602-e6be0344-b4e8-11e7-97fc-2601df8b0312.png)

Am I missing something?

Reply:
Bootstrap are using `$white` when it comes to that particular component. Thus, in the themebuilder it does not look as good. We could, perhaps, tie it to `$component` which is white.

Reply:
Fixed:
![image](https://user-images.githubusercontent.com/8048/31930227-b1500366-b8a7-11e7-94a1-4d33e6d220f2.png)